documentation/modules/payload/cmd/unix/bind_busybox_telnetd.md

```diff
@@ -21,6 +21,9 @@ with BusyBox telnetd installed.
   `pkill telnetd` and is used to avoid leaving a persistent command shell
   that does not require authentication.
 
+  **TelnatdPath**
```

Comment:
```suggestion
  **TelnetdPath**
```

Reply:
Not sure this is resolved? This file seems to have been removed from the change set?

Reply:
Resolved this in https://github.com/rapid7/metasploit-framework/pull/17255/commits/050c7a6ed3e2e6168850abce0a330fbd76bb2708